addons/mail/static/src/discuss/web/avatar_card/avatar_card_popover.xml

```diff
@@ -31,7 +31,7 @@
                 <div class="flex-wrap gap-2 mt-2">
                     <div class="justify-content-end d-flex align-items-baseline">
                         <div class="d-flex gap-2 o_avatar_card_buttons">
-                            <button t-if="!user.share" class="btn btn-secondary btn-sm" t-on-click.stop="onSendClick">Send message</button>
+                            <button t-if="!user.share and !props.hasNotSelfAsMember" class="btn btn-secondary btn-sm" t-on-click.stop="onSendClick">Send message</button>
```

Comment:
There is no such props, and it makes no sense to prevent sending a message to a specific user in certain context.